src/app/interactions/toggle/toggle-sample-2/toggle-sample-2.component.html

```diff
@@ -1,10 +1,8 @@
-<div class="sample-wrapper">
-  <article class="sample-column">
-    <button class="toggle-button"  igxButton="raised" [igxToggleAction]="toggleRef">Toggle</button>
+<div class="sample-column">
+    <button class="toggle-button" igxButton="raised" [igxToggleAction]="toggleRef">Toggle</button>
     <div class="toggle-content" igxToggle #toggleRef="toggle">
-      <section class="toggle-section">
-        <img src="assets/images/toggle/nature.jpg"/>
-      </section>
+        <section class="toggle-section">
+            <h5>Automatic toggle actions</h5>
```

Comment:
Can we change this h5 to something a bit smaller? An h6 perhaps? This still looks way too big.